commit 733e417518a69b71061c3bafc2bf106109565eee
Author: Wasin Thonkaew <wasin@wasin.io>
Date:   Wed Nov 3 19:42:08 2021 +0000

    asm-generic/error-injection.h: fix a spelling mistake, and a coding style issue
    
    Fix a spelling mistake "ganerating" -> "generating".
    Remove trailing semicolon for a macro ALLOW_ERROR_INJECTION to fix a
    coding style issue.
    
    Signed-off-by: Wasin Thonkaew <wasin@wasin.io>
    Acked-by: Masami Hiramatsu <mhiramat@kernel.org>
    Signed-off-by: Arnd Bergmann <arnd@arndb.de>

diff --git a/include/asm-generic/error-injection.h b/include/asm-generic/error-injection.h
index 7ddd9dc10ce9..fbca56bd9cbc 100644
--- a/include/asm-generic/error-injection.h
+++ b/include/asm-generic/error-injection.h
@@ -20,7 +20,7 @@ struct pt_regs;
 
 #ifdef CONFIG_FUNCTION_ERROR_INJECTION
 /*
- * Whitelist ganerating macro. Specify functions which can be
+ * Whitelist generating macro. Specify functions which can be
  * error-injectable using this macro.
  */
 #define ALLOW_ERROR_INJECTION(fname, _etype)				\
@@ -29,7 +29,7 @@ static struct error_injection_entry __used				\
 	_eil_addr_##fname = {						\
 		.addr = (unsigned long)fname,				\
 		.etype = EI_ETYPE_##_etype,				\
-	};
+	}
 
 void override_function_with_return(struct pt_regs *regs);
 #else